Flagsmith.FlagsmithClient/FlagsmithClient.cs

```diff
@@ -203,7 +203,7 @@ public async Task<IFlags> GetEnvironmentFlags()
 
         private async Task<IFlags> GetFeatureFlagsFromCorrectSource()
         {
-            return Environment != null ? GetFeatureFlagsFromDocument() : await GetFeatureFlagsFromApi().ConfigureAwait(false);
+            return OfflineMode && Environment != null ? GetFeatureFlagsFromDocument() : await GetFeatureFlagsFromApi().ConfigureAwait(false);
```

Comment:
Hmm... won't this break local evaluation mode as well? Shouldn't this be something like `(OfflineMode || LocalEvaluationEnabled) && ...`?

Reply:
You’re right, Done